CONTRIBUTING.md

```diff
@@ -1,155 +0,0 @@
-# Contributing to Chef
```

Comment:
I see that this file is replaced by a line in the `README.md`. That's probably fine, but Github treats the `CONTRIBUTING.md` as special and links to it when someone opens a PR, so I would keep it with the same line it in that is in the `README.md` pointing to chef/chef.